src/styles/dark.scss

```diff
@@ -64,6 +64,34 @@ body {
       color: #82b7f6;
     }
   }
+
+  aside.tip,
```

Comment:
just a NIT. You could have actually extracted the `aside` separately and nested `&.tip`. `&.warning` that way we could have moved all aside code to one place.